### PR TITLE
For v2.4.0 release

### PR DESCRIPTION
**Pull request by jeevatkm**

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
+golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3/go.mod h1:t9HGtf8HONx5eT2rtn7q6eTqICYqUVnKs3thJo3Qplg=
+golang.org/x/net v0.0.0-20201031054903-ff519b6c9102 h1:42cLlJJdEh+ySyeUUbEQ5bsTiq8voBeTuweGVkY6Puw=
+golang.org/x/net v0.0.0-20201031054903-ff519b6c9102/go.mod h1:sp8m0HH+o8qH0wwXwYZr8TS3Oi6o0r6Gce1SSxlDquU=
 golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a/go.mod h1:STP8DvDyc/dI5b8T5hshtkjS+E42TnysNCUPdjciGhY=
-golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
+golang.org/x/sys v0.0.0-20190412213103-97732733099d/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
+golang.org/x/sys v0.0.0-20200930185726-fdedc70b468f/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
 golang.org/x/text v0.3.0/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
+golang.org/x/text v0.3.3/go.mod h1:5Zoc/QRtKVWzQhOtBMvqHzDpF6irO9z98xDceosuGiQ=
+golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e/go.mod h1:n7NCudcB/nEzxVGmLbDWY5pfWTLqBcC2KZ6jyYvM4mQ=
```
Comment (moorereason) [inline go.sum:11]: Needs `go mod tidy`?

Comment (jeevatkm) [reply go.sum:11]: Actually, I ran tidy. Let me try again.

**Comment by moorereason** [comment]
@jeevatkm, when do you plan to release v2.4.0?

**Comment by jeevatkm** [comment]
@moorereason I'm sorry for the delay. It could have been done last Nov. Really my work hours are messed up (making comments around 5 am PST 🤣). 

I think it good to include PR #398 in this release. As soon as it's merged, I will make a release.

**Comment by jeevatkm** [comment]
@moorereason This PR ready for release - v2.4.0

**Review by moorereason — APPROVED** [review]
LGTM 🚀
